src/components/QuickGrid/QuickGrid.tsx

```diff
@@ -601,7 +601,7 @@ export class QuickGridInner extends React.Component<IQuickGridProps, IQuickGridS
         return (
             <div className={mainClass}>
                 <div className="hoverActions">
-                    <QuickGridRowContextActionsHandler ref={this._setRowContextActionsHandler} onGetRowActions={this.getRowContextActions} onActionClicked={this.onRowContextActionClicked} />
+                    <QuickGridRowContextActionsHandler ref={this._setRowContextActionsHandler} onGetRowActions={this.getRowContextActions} onActionClicked={this.onRowContextActionClicked} hideDropdownActionIcons={this.props.gridActions.hideDropdownActionIcons}/>
```

Comment:
in line with the comment about avoiding the dropdown hide icons property, this is also not needed

Reply:
on the second pass, i am wrong, this is actually needed, i mistakenly thought that the whole gridactions was passed down to the actionshandler. Live & learn.